runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -300,6 +300,10 @@ task cleanUpDockerJavaImages() {
 // task directly ('dependsOn buildAndPushDockerPythonContainer'). This ensures the correct
 // task ordering such that the registry doesn't get cleaned up prior to task completion.
 task buildAndPushDockerPythonContainer() {
+  if (!project.project(":sdks:python").buildFile.exists()) {
```

Comment:
What causes the build file to be missing?

Reply:
We don’t preserve github directory structure in google3. Java and Python imports have separate root directories so we can’t use Python build file from Java build file.